src/routes/safe/components/Apps/hooks/permissions/usePermissions.ts

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react'
+import local from 'src/utils/storage/local'
+
+const hasAnyKey = (obj: any) => !!Object.keys(obj).length
```

Comment:
This function cannot accept `any`, if you do `Object.keys(null)`, it will throw. I suggest changing it to `Record<string, unknown>`

Reply:
changed to `Record<string, any>` as cannot assign `unknown`  to the generic type in `usePermissions()`